RedisInternal.cpp

```diff
@@ -62,10 +63,45 @@ void RedisArray::init(Client &client)
 
 RedisArray::operator std::vector<String>() const
 {
-    std::vector<String> rv;
-    for (auto ro : vec)
-        rv.push_back((String)*ro.get());
-    return rv;
+    std::vector<String> result;
```

Comment:
You implemented nested handling here, too?! Huge bonus implementation, you rock @nafraf thank you!! 🚀 🙇